src/medal_app/views.cljs

```diff
@@ -19,29 +19,29 @@
 
 (defn medal-body []
   (let [loading? (re-frame/subscribe [::subs/loading?])
-        medals (re-frame/subscribe [::subs/medals])
-        sort-order (re-frame/subscribe [::subs/sort-order])]
+        sort-order (re-frame/subscribe [::subs/sort-order])
+        sorted-medals (re-frame/subscribe [::subs/sorted-medals])]
```

Comment:
you can deref the subscription here and not have to repeat using the `@` below.
```suggestion
        sort-order @(re-frame/subscribe [::subs/sort-order])
        sorted-medals @(re-frame/subscribe [::subs/sorted-medals])]
```

otherwise these variables should be prefixed with a `*`

```suggestion
       *sort-order (re-frame/subscribe [::subs/sort-order])
        *sorted-medals (re-frame/subscribe [::subs/sorted-medals])]
```
anytime we store an atom (which is what re-frame/subscribe returns) to a variable we like to name that variable starting with a `*` to make it clear. It's helpful when using in conditionals `(if somevar? ... ...)` if it's an atom currently in a false state it will still return true if we forgot to deref. `(if *somevar? ... ...)` makes it easier to catch the mistake.